examples/client-logger/app-insights-logger/package.json

```diff
@@ -17,6 +17,7 @@
 	"scripts": {
 		"build": "fluid-build . --task build",
 		"build:compile": "fluid-build . --task compile",
+		"build:test": "tsc --project tsconfig.json",
```

Comment:
Is this necessary? This seems unrelated to type-tests.

Reply:
I thought this is the type test... Adding test class here